gulpfile.js

```diff
@@ -87,9 +87,16 @@ const buildWasmTask = createTask({
       GOARCH: 'wasm',
     }
 
-    const command = `go build -o "${wasmOutputPath}" main.go`
+    exec('cp "$(go env GOROOT)/lib/wasm/wasm_exec.js" ./src/js/vendor', { cwd: __dirname }, (err, stdout, stderr) => {
+      if (err) {
+        log.error('Error copying wasm_exec.js:', stderr)
+        done(err)
+        return
+      }
+      done()
+    })
```

Comment:
I'm not sure if this is appropriate, but, if it makes sense, then maybe `/src/js/vendor/wasm_exec.js` should be deleted and added to `.gitignore`?

Reply:
PS: It looks like we're still installing GO_VERSION=1.23.4 for some reason: https://app.netlify.com/sites/docs-ui/deploys/682252d758947e00081e45f3. The new version requires at least 1.24.1. Not sure how to update Netlify